### ISSUE TITLE
Add icon for amenity=animal_boarding

### ISSUE THREAD

**Issue by Adamant36** [gravitystorm/openstreetmap-carto#3966]
It would be cool if there was an icon for animal boarding facilities. As it is their name's are already rendered in a generic text label color. So it wouldn't be that much of a leap to also add an icon. My suggestion is going with paw icon of veterinary, but without the cross in the middle. Since there is some overlap there and it makes sense.

**Comment by HolgerJeromin** [comment]
> As it is their name's are already rendered in a generic text label color.

There is no such thing since a many years.
Can you add a screenshot and a link to a location?

**Comment by Adamant36** [comment]
@HolgerJeromin, yes, it is when the amenity is combined with a building, but ultimately that really shouldn't matter. If there is an amenity attached to the building the name of the amenity shouldn't render the same as a building name. Since they are separate objects IMHO. 

> We have 720 nodes and 1195 ways with this "pet hotel" feature.

That seems close enough to the threshold to me. There's actually about a thousand more uses of amenity=animal_shelter. Which I find interesting.

**Comment by jeisenbe** [comment]
The tag [amenity=animal_boarding](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Danimal_boarding) is slightly less common than the two other animal-facility tags which were approved around the same time: [amenity=animal_shelter](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Danimal_shelter) and [amenity=animal_breeding](https://wiki.openstreetmap.org/wiki/Tag:amenity%3Danimal_breeding)

### Numbers:
animal_boarding - [1933 uses (taginfo)](https://taginfo.openstreetmap.org/tags/?key=amenity&value=animal_boarding#overview)
animal_breeding - [2 841 uses ](https://taginfo.openstreetmap.org/tags/?key=amenity&value=animal_breeding#overview)
animal_shelter - [3002 uses[(https://taginfo.openstreetmap.org/tags/?key=amenity&value=animal_shelter#overview)

### Maps: 
Animal shelter is more common in North America and Australia, animal breeding is more limited in extent. Here in Indonesia animal_boarding is the only one of the 3 that is common. 

animal_boarding:
![animal_boarding-map](https://user-images.githubusercontent.com/42757252/68363439-3080d000-016e-11ea-82ec-b7fbda24c7fa.png)

animal_breeding:
![animal_breeding-map](https://user-images.githubusercontent.com/42757252/68363462-3c6c9200-016e-11ea-9902-9053ae457192.png)

animal_shelter:
![animal_shelter-map](https://user-images.githubusercontent.com/42757252/68363467-4098af80-016e-11ea-9df8-566c7b178018.png)

### History chart
![taghistory-animal-shelter-breeding-boarding](https://user-images.githubusercontent.com/42757252/68363701-1dbacb00-016f-11ea-8ca3-c4f6a0566a93.png)

Usage is growing for all (not sure why the dotted line for animal_shelter isn't available, but it should meet up at 3000)

But these are still only the 98th, 99th and 125th most common values of amenity: https://taginfo.openstreetmap.org/keys/amenity#values - there are a number of more common features which are not yet rendered in this style.  

For example, amenity=watering_place is also related to animals (a place where "animals such as horses can drink") and is used 7371 times, but not yet rendered.

**Comment by Adamant36** [comment]
Here's the graph for amenity=animal_shelter by itself. Seems like it's on a good, consistent upward path. 

![taghistory (5)](https://user-images.githubusercontent.com/30259065/68434870-e2b89780-016e-11ea-8330-7f2e213f046b.png)

> there are a number of more common features which are not yet rendered in this style.

Obviously. My main impetuous for favoring rendering of this is that the names already render when they tagged on buildings and my guess is that the vast majority of them are. So it wouldn't be a "new object" being added to the map per say. I'd perfectly fine with just a dot and label rendering (by itself if need be) in a unique color if nothing else. Then people will at least know the name is related an amenity and not just the name of the building (that could go for all three tags or just amenity=animal_boarding. I don't really care).

**Comment by Adamant36** [comment]
> For example, amenity=watering_place is also related to animals (a place where "animals such as horses can drink") and is used 7371 times, but not yet rendered.

@jeisenbe, amenity=watering_place was declined by @matkoniecz in #3703, 8 months before your comment. Perhaps you could revive it. As his reason for the decline wasn't clear and @imagico said it might possibly be rendered if #1224 is worked out.

**Comment by vbertola** [comment]
+1 for rendering animal_*, especially animal_shelter, especially when they are areas. We have the municipal dog pound here:

https://www.openstreetmap.org/way/827434380#map=18/45.12975/7.70202

and there is no way to find it on the map because it is not rendered at all - only the various buildings it is made of. Now, I could assign the "dog pound" name to one (or each) of the buildings, but that would be tagging for the renderer... The clean solution is just to draw the area and render the name over it.

**Comment by sun-geo** [comment]
> ... Now, I could assign the "dog pound" name to one (or each) of the buildings, but that would be tagging for the renderer...

@vbertola 

I wouldn't do that.
Maybe there is another possible workaround, in meantime untill carto does a rendering for  `amenity=animal_boarding`, to get the name displayed, by adding an [office=*](https://wiki.openstreetmap.org/wiki/Key:office) tag, if there is really an office on site.
Btw, what kind of institution is it (w827434380)? For an association there is a potentially `office=association` available, or it will be provided from government (municipality) then maybe `office=government` + `government=animal_welfare` could suit.

**Comment by Adamant36** [comment]
Or in the meantime use landuse=commerical with whatever animal tag. It wouldn't be 100% correct, but it wouldn't be totally wrong either.
